olive/passes/onnx/optimum_conversion.py

```diff
@@ -36,6 +36,7 @@ def _run_for_config(
 
         from optimum.exporters.onnx import main_export as export_optimum_model
 
+        model.extend_model_attributes_from_hf_config()
```

Comment:
>extend_model_attributes_from_hf_config [](http://example.com/codeflow?start=14&length=38)

this call should be part of model.__init__?

Reply:
We cannot put it under model.init as the input model can be in remote path where need be downloaded firstly.